### ISSUE TITLE
Node/Angular Promise catch method error "missing space before ("

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google/closure-linter#85]
```
What steps will reproduce the problem?
1. Use promise chaining method
2. On promise chain call catch method to handle errors

somePromiseFunc('foo')
   .then(function(result) {
      /*handle result */
   })
   .catch(function(err) {
      /* handle error */
   });

What is the expected output? What do you see instead?

Should give no error. Google linter reports "missing space before (" on catch 
line.

What version of the product are you using? On what operating system?

2.3.13, OSX 10.9.5

Please provide any additional information below.


```

Original issue reported on code.google.com by `jn.marti...@gmail.com` on 5 Dec 2014 at 6:38

**Comment by haynesgt** [comment]
Issue observed in v2.3.13, with both `catch` and `finally`.
Issue not observed in v2.3.19.
`finally` a method is tested in v2.3.19:closure_linted/testdata/spaces.js:327
`catch` as a method is not tested
